x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/authc/ApiKeyIntegTests.java

```diff
@@ -1171,6 +1303,92 @@ public void testApiKeyAuthorizationApiKeyMustBeAbleToRetrieveItsOwnInformationBu
         assertErrorMessage(ese, "cluster:admin/xpack/security/api_key/get", ES_TEST_ROOT_USER, responses.get(0).getId());
     }
 
+    public void testApiKeyViewLimitedBy() {
```

Comment:
Optional, but wondering if we should randomize between query and get in this method -- would give a nice single place where we test both ways to view limited by

Reply:
Yep, added the randomisation.